src/main/java/mate/academy/bookstore/config/EnvConfig.java

```diff
@@ -0,0 +1,14 @@
+package mate.academy.bookstore.config;
+
+import io.github.cdimascio.dotenv.Dotenv;
+import jakarta.annotation.PostConstruct;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class EnvConfig {
+    @PostConstruct
+    public void init() {
+        Dotenv dotenv = Dotenv.load();
```

Comment:
If you need to get properties from env file, you can use in application.properties
`spring.config.import=optional:file:.env[.properties]`